apps/explorer/src/pages/transaction-result/TransactionData.tsx

```diff
@@ -0,0 +1,279 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
it feels like there's enough going on here to warrant splitting out some of these components into separate files

Reply:
Will do this in follow up PR, ive tracked this suggestion